projects/zookeeper.cmake

```diff
@@ -142,7 +142,7 @@ function(build_zookeeper)
     xpNewDownload(${CPP_UNIT})
 
     set(ACLOCAL_STR "aclocal -I ${SOURCE_DIR}/src/c/cppunit-${CPP_UNIT_VER}")
-    ExternalProject_Add(zookeeper_configure DEPENDS zookeeper download_cppunit-${CPP_UNIT_VER}.tar.gz
+    ExternalProject_Add(zookeeper_configure DEPENDS zookeeper_ant download_cppunit-${CPP_UNIT_VER}.tar.gz
```

Comment:
To be honest, I still haven't tried to build everything. I'm going to let it build overnight, starting in 30 minutes or so. I'll post the results, probably late tomorrow night.